features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -24,19 +24,10 @@ Feature: cluster monitoring related upgrade check
     Given I switch to cluster admin pseudo user
     And I use the "openshift-monitoring" project
     Given all pods in the project are ready
-    Then the step should succeed
-    When I run the :get client command with:
-      | resource      | clusteroperator |
-      | resource_name | monitoring      |
-    And evaluation of `@result[:stdout].split(/\n/)[1].split(/\s+/)[4]` is stored in the :degreaded_status clipboard
-    Then the expression should be true> cb.degreaded_status == "False"
+    And the expression should be true> cluster_operator('monitoring').condition(type: 'Degraded')['status'] == "False"
 
     #check retention time
-    When I run the :get client command with:
-      | resource      | prometheus |
-      | resource_name | k8s        |
-      | o             | yaml       |
-    Then the expression should be true> YAML.load(@result[:stdout])["spec"]["retention"] == "3h"
+    Given the expression should be true> prometheus('k8s').retention == "3h"
```

Comment:
suppose should be changed as prometheus('k8s').retention(cached: false) == "3h"

Reply:
`cached:false` should be used only for values that change frequently. E.g, status.

Reply:
By default the retention is a different value, I just configure it as 3h and check the value, not sure it will get the new value.

Reply:
Feel free to raise a PR to fix that if it uses the cached value wrongly.

Reply:
It should be ok if no other tests read the retention before I change configuration.